holidays/countries/ivory_coast.py

```diff
@@ -0,0 +1,95 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+from gettext import gettext as tr
+
+from holidays.constants import PUBLIC
+from holidays.groups import ChristianHolidays, IslamicHolidays, InternationalHolidays
+from holidays.observed_holiday_base import ObservedHolidayBase, SUN_TO_NEXT_MON
+
+
+class IvoryCoast(ObservedHolidayBase, ChristianHolidays, IslamicHolidays, InternationalHolidays):
+    """
+    Ivory Coast holidays.
+
+    References:
+        * <https://natlex.ilo.org/dyn/natlex2/r/natlex/fe/details?p3_isn=44374>
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_Ivory_Coast>
+        * <https://www.timeanddate.com/holidays/ivory-coast/>
+        * <https://holidayapi.com/countries/ci/>
+    """
+
+    country = "CI"
+    estimated_label = tr("%s (estimated)")
+    observed_label = tr("%s (observed)")
+    observed_estimated_label = tr("%s (observed, estimated)")
+    supported_categories = (PUBLIC,)
+    # Ivory Coast gained independence in 1960.
+    start_year = 1960
+
+    def __init__(self, islamic_show_estimated: bool = True, *args, **kwargs):
+        """
+        Args:
+            islamic_show_estimated:
+                Whether to add "estimated" label to Islamic holidays name
+                if holiday date is estimated.
+        """
+        ChristianHolidays.__init__(self)
+        IslamicHolidays.__init__(self, show_estimated=islamic_show_estimated)
+        InternationalHolidays.__init__(self)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # Secular Holidays
+        # New Year's Day.
+        self._add_observed(self._add_new_years_day(tr("New Year's Day")), rule=SUN_TO_NEXT_MON)
```

Comment:
```suggestion
        self._add_new_years_day(tr("New Year's Day"))
```
In the law, I see the shift of the day off to Monday only for Labor Day and National Peace Day.

Reply:
Ah right, my bad :sweat_smile: